documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -237,14 +237,18 @@ include::modules/new-migrating-virtual-machines-cli.adoc[leveloffset=+3]
 :rhv:
 include::modules/new-migrating-virtual-machines-cli.adoc[leveloffset=+3]
 :rhv!:
-:context: ova
-:ova:
-include::modules/new-migrating-virtual-machines-cli.adoc[leveloffset=+3]
-:ova!:
```

Comment:
@RichardHoch - Changing the order. Remember that it is not just opening the condition but also closing the condition, otherwise you will get a build error 

![image](https://github.com/kubev2v/forklift-documentation/assets/106804821/d025c9f0-f729-476f-ac7b-683b95e1c935)